src/main.js

```diff
@@ -103,11 +113,17 @@ routeConfig(app, [
   new WebhookController(settings.webhookRepository),
 ]);
 
-app.use((request, response, next) => response.sendStatus(404));
```

Comment:
do we need this middleware at all in main.js ? or we need to move it to RouteConfig.js somehow?